content/community/changelog/app-frontend/v3/whats-new/_index.nb.md

```diff
@@ -3,6 +3,39 @@ title: Hva er nytt?
 description: Oversikt over endringer som ble introdusert i v3 av app frontend.
 toc: true
 ---
+## 3.27.0 (2022-02-01) - Vise app eier i header
+App eier vil nå vises i header for å tydeliggjøre hvem som er eier av appen.
+
+Endringen støtter under sak [#7227](https://github.com/Altinn/altinn-studio/issues/7227).
```

Comment:
```suggestion
Issue [#7227](https://github.com/Altinn/altinn-studio/issues/7227).
```

For å være konsekvent med navngivning tidligere for saker som løser hele issues.